### PR TITLE
Starter kit docs & guide

### PR DESCRIPTION
**Pull request by dafuga**

### CODE REVIEW

File: src/lib/guides/StarterKits/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+requiresReview: true
+---
+
+# Introduction to Wharfkit Starter Kits
+
+The Starter kits are a great way to start developing applications on Wharfkit. They offer a convenient way to install all the essential Wharfkit packages at once, while also providing `sessionKit` instances that can be used quickly instantiate sessions in your applications.
+
+The available kits are `@wharfkit/starter`, `@wharfkit/starter-eos`, `@wharfkit/starter-telos`, `@wharfkit/starter-wax`, and `@wharfkit/starter-jungle`.
+
+> **NOTE**: This guide is current as of December 2023 and reflects the latest versions of Wharfkit Starter Kits.
```
Comment (aaroncox) [inline src/lib/guides/StarterKits/getting-started.md]: I don't think this is needed, this was included in other guides when we were pre-release.

File: src/lib/guides/StarterKits/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Getting Started
+
+Ensure [Node.js](https://nodejs.org/en/) is installed, as it is a prerequisite for using Wharfkit packages.
```
Comment (aaroncox) [inline src/lib/guides/StarterKits/getting-started.md]: NPM or Yarn is required, but not nodejs specifically? 🤔 

I'm not sure we even need this section TBH. People could be using bun too

File: src/lib/guides/StarterKits/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+### Step 3: Using SessionKit
+
+Use the `sessionKit` instance to log the user in and create a Wharf session:
+
+```js
+import { sessionKit } from '@wharfkit/starter';
+
+const session = await sessionKit.login(); // Initiates a login request
+```
```
Comment (aaroncox) [inline src/lib/guides/StarterKits/getting-started.md]: This is lacking a lot of information, but perhaps we could link to one of the other guides to fill in the gaps?

More detail should probably be included here that explains how its already setup with API endpoints and plugins. We can then link off to other sections for more information.

File: src/lib/guides/StarterKits/getting-started.md (hunk 2 above)
Comment (dafuga) [reply src/lib/guides/StarterKits/getting-started.md]: Not sure that we need it either, but can you even download npm or yarn without Node.js? 😄

File: src/lib/guides/StarterKits/getting-started.md (hunk 3 above)
Comment (dafuga) [reply src/lib/guides/StarterKits/getting-started.md]: Yeah, that makes sense!

**Comment by aaroncox** [comment]
Do we need a pass over this to update now that the starter kits are effectively released? 

I see it mentions the account kit, which AFAIK isn't included anymore. I'm just wondering if there's more things like this.

File: src/lib/docs/Utilities/StarterKits.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+# or
+npm install --save @wharfkit/starter-jungle
+```
+
+## Usage
+
+Once installed, you can start importing and using the core Wharfkit modules that you need in your project. For example:
+
+```ts
+import {SessionKit, ContractKit, AccountKit} from '@wharfkit/session';
```
Comment (aaroncox) [inline src/lib/docs/Utilities/StarterKits.md]: These are wrong now, yeah?

Comment (dafuga) [reply src/lib/docs/Utilities/StarterKits.md]: yep 😅

Comment (dafuga) [reply src/lib/docs/Utilities/StarterKits.md]: fixed!

**Pull request merged**